components/ContributorCard.js

```diff
@@ -86,7 +86,9 @@ const ContributorCard = ({ intl, width, height, contributor, currency, ...props
           </H5>
         </LinkContributor>
         <StyledTag my={2} padding="5px" letterSpacing="0.05em" fontStyle="initial">
-          {formatMemberRole(intl, getMainContributorRole(contributor))}
+          {getMainContributorRole(contributor) == 'BACKER' && contributor.totalAmountDonated < 1
```

Comment:
We have a string constant for this, it's better to use it

```suggestion
          {getMainContributorRole(contributor) === roles.BACKER && contributor.totalAmountDonated < 1
```